src/PrestaShopBundle/Form/Admin/Sell/Product/SEO/SEOType.php

```diff
@@ -78,20 +84,28 @@ public function __construct(
         RouterInterface $router,
         bool $friendlyUrlEnabled,
         bool $forceFriendlyUrl,
-        LegacyContext $legacyContext
+        LegacyContext $legacyContext,
+        ConfigurationInterface $configuration
     ) {
         parent::__construct($translator, $locales);
         $this->router = $router;
         $this->friendlyUrlEnabled = $friendlyUrlEnabled;
         $this->forceFriendlyUrl = $forceFriendlyUrl;
         $this->legacyContext = $legacyContext;
+        $this->configuration = $configuration;
     }
 
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        // Automatic update is only enabled when product is offline and the configuration is enabled
```

Comment:
Should be a catalog config var, something like `PS_FORCE_FRIENDLY_PRODUCT_ACTIVE_ONLY` ?

should we also take into account the fields like `visibility`, `on_sale`, and `available_for_order` ?